plugin/src/main/resources/io/jenkins/plugins/analysis/core/model/RunTab/index.jelly

```diff
@@ -0,0 +1,65 @@
+<!--
+The MIT License
+
+Copyright (c) 2025, Jan Faracik
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+  <l:run-subpage>
+    <st:adjunct includes="io.jenkins.plugins.analysis.core.model.RunTab.styles"/>
+
+    <l:app-bar title="${it.displayName}" />
+
+    <div style="display: grid; grid-template-columns: 300px 1fr; gap: var(--section-padding)">
+      <div id="side-panel" class="app-page-body__sidebar wng-sidebar">
+        <l:tasks>
+          <j:forEach var="item" items="${it.warningActions}" indexVar="index">
+            <l:task title="${item.displayName}"
+                    icon="${item.iconFileName}"
+                    href="${index eq 0 ? url + '/warnings' : item.urlName}"
+                    badge="${item.badge}" />
+          </j:forEach>
+        </l:tasks>
+      </div>
+      <div>
+        <j:set var="it" value="${resultAction.target}" />
+
+        <st:adjunct includes="io.jenkins.plugins.echarts"/>
+        <link rel="stylesheet" href="${resURL}/plugin/warnings-ng/css/custom-style.css"/>
+
+        <j:set var="i" value="${it.issues}"/>
+
+        <j:choose>
+          <j:when test="${i.isNotEmpty()}">
+            <st:include page="filled.jelly" it="${it}" />
+          </j:when>
+          <j:otherwise>
+            <st:include page="empty.jelly" it="${it}" />
+          </j:otherwise>
+        </j:choose>
```

Comment:
The links in the tabs (to the source code or to the package drill down) do not work anymore. 

Maybe I need to rethink how the warnings are currently shown...